packages/ra-ui-materialui/src/button/ExportButton.tsx

```diff
@@ -92,10 +92,8 @@ const defaultIcon = <DownloadIcon />;
 const sanitizeRestProps = ({
     resource,
     ...rest
-}: Omit<
-    ExportButtonProps,
-    'sort' | 'maxResults' | 'label' | 'exporter' | 'meta'
->) => rest;
+}: Omit<ExportButtonProps, 'maxResults' | 'label' | 'exporter' | 'meta'>) =>
```

Comment:
**suggestion**: Since `ExportButtonProps` does not contains the `sort` prop, we could remove this from `ExportButton.propTypes` as well

Reply:
Right, fixed